src/pages/core/plugins/freeze-execute.md

```
@@ -0,0 +1,513 @@
+
+## Important Notes
+
+- When the `frozen` field is set to `true`, any execute operations will be blocked
+- **Default authority**: The asset owner controls the plugin by default
+- **Authority delegation**: Only the current authority can freeze/unfreeze the execute functionality
+- **Authority constraints**: If authority is delegated to someone else, the original owner cannot unfreeze until authority is revoked
+- The plugin cannot be removed when frozen
+- Authority cannot be reassigned when frozen
+- The plugin works with the [Execute instruction](/core/execute-asset-signing) system
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)